app/controllers/questions_controller.rb

```diff
@@ -90,8 +90,7 @@ def recently_commented
       .where(status: 1)
     @pagy, @questions = pagy(filter_questions_by_tag(@questions, params[:tagnames])
       .joins(:comments)
-      .order('comments.timestamp DESC')
-      .group('node.nid'), items: 24)
```

Comment:
I have removed the `group` function from here for two reasons:

1. The `@questions` we are filtering on are already being grouped by `node.nid`.
2. The MySQL issue that is coming up was that ordering was happening on a variable on which grouping is not happening.

I believe that because of this, the behaviour of the code should not change, and we should get the same results.